features/src/rapids-build-utils/opt/rapids-build-utils/manifest.yaml

```
@@ -78,9 +78,13 @@ repos:
       sub_dir: cpp/libcudf_kafka
       depends: [cudf]
   python:
+    - name: pylibcudf
+      sub_dir: python/pylibcudf
+      depends: [cudf]
+      args: {cmake: -DFIND_CUDF_CPP=ON, install: *rapids_build_backend_args}
     - name: cudf
       sub_dir: python/cudf
-      depends: [cudf]
+      depends: [cudf, pylibcudf]
```

Don't need to modify this line, as this indicates a dependency on the C++ `cudf` library:

```suggestion
      depends: [cudf]
```